rep-0003.rst

```diff
@@ -220,7 +220,62 @@ Exact or Series Requirements:
 - Qt 5.3.x
 - PyQt5
 
-Buildsystem support:
+Build System Support:
+
+- catkin:
+
+  - build from source
+  - release for binary packaging
+  - wiki documentation
+  - continuous integration
+
+- rosbuild:
+
+  - build from source
+
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 25
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+ The following requirements are the same as Kinetic's to avoid maintainers
+ to branch out before the next LTS release
+
+- C++11
+- Python 2.7
+
+  - Python 3.5 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+- Boost 1.55
+
+Exact or Series Requirements:
+
+- Ogre3D 1.9.x
+- Gazebo 8
+- PCL 1.7.x
+- OpenCV 3.1.x
+- Qt 5.3.x
+- PyQt5
+
+Build System Support:
```

Comment:
Good point, fixed in https://github.com/ros-infrastructure/rep/pull/123/commits/42fa94534fd2a16df1ad004f60a59b0f79c2511b